src/Components/v2/CollectionsHubsNav.tsx

```diff
@@ -1,32 +1,19 @@
 import { CSSGrid, Serif } from "@artsy/palette"
 import { CollectionsHubsNav_marketingHubCollections } from "__generated__/CollectionsHubsNav_marketingHubCollections.graphql"
+import { useTracking } from "Artsy/Analytics"
 import * as Schema from "Artsy/Analytics/Schema"
-import { useTracking } from "Artsy/Analytics/useTracking"
-import React, { FC, useEffect } from "react"
+import React, { FC } from "react"
 import { createFragmentContainer, graphql } from "react-relay"
 import { resize } from "Utils/resizer"
 import { ImageLink } from "./ImageLink"
 
 interface CollectionsHubsNavProps {
   marketingHubCollections: CollectionsHubsNav_marketingHubCollections
-  collectionHubTestVariation: string
 }
 
 export const CollectionsHubsNav: FC<CollectionsHubsNavProps> = props => {
   const { trackEvent } = useTracking()
 
-  useEffect(() => {
-    const experiment = "collection_hub_entrypoints_test"
-    trackEvent({
-      action_type: Schema.ActionType.ExperimentViewed,
-      experiment_id: experiment,
-      experiment_name: experiment,
-      variation_id: props.collectionHubTestVariation,
-      variation_name: props.collectionHubTestVariation,
-      nonInteraction: 1,
-    })
-  }, [])
-
```

Comment:
When we fired the Experiment Viewed event here and it was only getting called for the experiment and excluding the control. We want it to be called for both so we moved the call higher up (i.e. in the  Collect component).